script/options.js

```diff
@@ -2,14 +2,11 @@
 
 const path = require('path');
 const commandLineArgs = require('command-line-args');
-const applyHerokuOptions = require('./heroku-helper');
 const environments = require('./constants/environments');
 const hostnames = require('./constants/hostnames');
 
 const COMMAND_LINE_OPTIONS_DEFINITIONS = [
   { name: 'buildtype', type: String, defaultValue: environments.LOCALHOST },
-  { name: 'brand-consolidation-enabled', type: Boolean, defaultValue: false },
-  { name: 'no-sanity-check-node-env', type: Boolean, defaultValue: false },
```

Comment:
I removed this `no-sanity-check-node-env` because it never made much sense to me.